lib/github_changelog_generator/generator/entry.rb

```diff
@@ -123,6 +123,7 @@ def generate_body(pull_requests, issues)
     # @return [Array] Section objects.
     def default_sections
       [
+        Section.new(name: "summary", prefix: "", labels: @options[:summary_labels], options: @options, body_only:true),
```

Comment:
Would you mind adding an `@options[:summary_prefix]` option that just defaults to `""`? I actually really like the idea of summaries being under the version H2 line, but currently my company adds release summaries with a `### Summary` prefix when writing release changelogs by hand. Eg: https://github.com/puppetlabs/puppetlabs-apache/blob/master/CHANGELOG.md

Reply:
+1 here. I'm not biased or anything, though.

Reply:
That makes sense.
I will add it option later :)

---
- **--summary-label [LABLE]**

Set up custom label for release summary section.

Default: **''**